### ISSUE TITLE
Using LaserSDK in sub-threads

### ISSUE THREAD

**Issue by EBenkler** [toptica/python-lasersdk#7]
I tried to access a DLC Pro via the Laser SDK, when the calls are made form a different thread than the main thread. I received this runtime error:
`There is no current event loop in thread`

Is this the intended behaviour? As a workaround, I geberate a new event loop in the thread, first. 

See this minimal working example when calling Laser SDK clients from the main thread, from a sub thread with generation of an event loop inside the thread and from a sub thread without generation of a new event loop:

```
from toptica.lasersdk.client import Client, NetworkConnection, UserLevel
from threading import Thread
import asyncio

def set_A_out():
    DLCPro_IP = "c21153" # substitute by network name or IP address of DLCPro
    try:  
        DLC_client = Client(NetworkConnection(DLCPro_IP))
        DLC_client.open()  
        DLC_client.change_ul(UserLevel.MAINTENANCE, 'CAUTION')
        DLC_client.set('buzzer:sounds:on-connection',False)
        DLC_client.change_ul(UserLevel.NORMAL)
        DLC_client.set('io:out-a:voltage-set',0)
        DLC_client.close()
        print('success!')
    except Exception as e:
        print("Communication error with 633nm Laser Driver: " + str(e))

def with_new_loop():
    loop = asyncio.new_event_loop()
    asyncio.set_event_loop(loop)
    set_A_out()


print('execute in main thread:')
set_A_out()

print('execute in sub thread with generation of a new event loop:')
t1=Thread(target=with_new_loop)
t1.start()
t1.join() #wait for thread t1 to finish

print('execute in sub thread with new event loop:')
t2=Thread(target=set_A_out)
t2.start()
```

The output is:
```
execute in main thread:
success!
execute in sub thread with generation of a new event loop:
success!
execute in sub thread with new event loop:
Communication error with 633nm Laser Driver: There is no current event loop in thread 'Thread-44'.
```

Tested with Laser SDK 3.0.1, python 3.10 on Windows 10

**Comment by thtrummer** [comment]
Hello,

thank you for the issue. 

This is in fact not the intended behavior. It was caused by trying to simplify the type hints in the connection classes; the event loop isn’t even used.

The fix still has to make its way through the nightly tests, but we will make a new release early next week.

Best regards,
Thomas

**Comment by thtrummer** [comment]
Hello,

we have released a new version (3.1.1) that fixes the problem. You can update via

``pip install toptica_lasersdk --upgrade``

Thanks again for your report and please let us know if there are any further issues.

Best regards,
Thomas

**Comment by thtrummer** [comment]
Hello,

we are closing this issue for now. Feel free to create a new issue if there are any further problems.

Best regards,
Thomas
